go/runtime/history/prune.go

```diff
@@ -120,18 +120,14 @@ func (p *keepLastPruner) Prune(latestRound uint64) error {
 				break
 			}
 
-			if err := tx.Delete(roundResultsKeyFmt.Encode(round)); err != nil {
```

Comment:
I would clear any stored round results on startup once, since old round results will never be removed from db and will be kept forever. Furthermore, I would prepare a PR which will remove this cleanup and will be merged in some future version of Oasis Core.